### PR TITLE
docs: update license information

### LINKED ISSUE #3: Update copyright information
I think we should update copyright information in README and LICENSE by replacing author names with something like **Paymentsds Authors**, after that we can create an AUTHORS file to identify everyone who has contributed to the project

### PR DESCRIPTION
**Pull request by thatfiredev**
This should resolve #3

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (edsonmichaque) [inline LICENSE]: We should replace this template with the respective copyright information

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Anísio Mandlate
+Edson Michaque
+Elton Laice
+Nélio Macombo
+Rosário P. Fernandes
```
Comment (edsonmichaque) [inline AUTHORS]: Maybe we should also put contact information here, like:
...
Edson Michaque < edson@michaque.com >
...

**Review by edsonmichaque — CHANGES_REQUESTED** [review]
@rosariopfernandes, after improving what I've marked, we are read to merge

File: LICENSE (hunk 1 above)
Comment (thatfiredev) [reply LICENSE]: @edsonmichaque you sure? I copied this from our [JS SDK](https://github.com/paymentsds/mpesa-js-sdk/blob/develop/LICENSE), so maybe we need to change it there too?

File: AUTHORS (hunk 2 above)
Comment (thatfiredev) [reply AUTHORS]: Sounds good. Can you share your email contact? I'll ask the other authors as well.

File: LICENSE (hunk 1 above)
Comment (edsonmichaque) [reply LICENSE]: Yeah, we need to change that also

File: AUTHORS (hunk 2 above)
Comment (edsonmichaque) [reply AUTHORS]: Mine is edson@michaque.com

Comment (edsonmichaque) [reply AUTHORS]: We can ask them to make the change themselves :-D
So that they become **real** authors

Comment (thatfiredev) [reply AUTHORS]: @edsonmichaque hehe, makes sense to me!

**Pull request merged**